CMakeLists.txt

```diff
@@ -200,6 +200,9 @@ if(BUILD_GSTREAMER_PLUGIN)
 
   add_executable(kinesis_video_gstreamer_audio_video_sample_app samples/kinesis_video_gstreamer_audio_video_sample_app.cpp)
   target_link_libraries(kinesis_video_gstreamer_audio_video_sample_app ${GST_APP_LIBRARIES} KinesisVideoProducer)
+
+  add_executable(kinesis_file_gstreamer_sample_app samples/kinesis_file_gstreamer_sample_app.cpp)
```

Comment:
Need to think about the naming strategy. Prefixing everything with kinesis_video_streams_ is way to long.

Let's use kvs_

in this case it will be kvs_gstreamer_file_upload_sample

Obviously, you are free to chose a better naming.

Reply:
yea! I definitely feel the same way.